source/sass/31-molecules/_molecules.scss

```diff
@@ -0,0 +1,22 @@
+@forward 'authentication/authentication';
+@forward 'breadcrumb/breadcrumb';
+@forward 'building-blocks/primary-tabs';
+@forward 'content-moderation/content-moderation';
+@forward 'facet-range-slider/facet-range-slider';
+@forward 'facets/facets';
+@forward 'feature-block/feature-block';
+@forward 'fields/fields';
+@forward 'fields/multiple';
+@forward 'file-upload/file-upload';
+@forward 'filter-placeholder/filter-placeholder';
+@forward 'language-switcher/language-switcher';
+@forward 'local-tasks/local-tasks';
+@forward 'map/map';
+@forward 'modal/modal';
+@forward 'newsletter/newsletter';
+@forward 'oidc-menu/oidc-menu';
+@forward 'partner-block/partner-block';
+@forward 'search/search';
+@forward 'status-message/status-message';
+@forward 'teaser/teaser';
```

Comment:
@use should be enough here.

See https://mr-alien.medium.com/use-forward-in-sass-70b9cd44218b